pkg/backend/display/tree_test.go

```diff
@@ -205,7 +205,7 @@ func TestTreeRenderCallsFrameOnTick(t *testing.T) {
 	terminalText := buf.String()
 	assert.Contains(t, terminalText, "pulumi:pulumi:Stack")
 	assert.Contains(t, terminalText, "System Messages")
-	assert.Contains(t, terminalText, strings.Repeat("a", 1000))
+	assert.Contains(t, terminalText, strings.Repeat("a", 70))
```

Comment:
Why did this change?

Reply:
Because I fixed the renderer to trim lines, I can "unfix" if you would prefer

Reply:
No that's fine, easier if you pull little changes like that out to their own but no need now.